contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -94,6 +94,7 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
   mapping(address => Signer) internal s_signers;
   address[] internal s_signersList; // s_signersList contains the signing address of each oracle
   address[] internal s_transmittersList; // s_transmittersList contains the transmission address of each oracle
+  address[] internal s_deactivatedTransmittersList;
```

Comment:
we might want to make this an `EnumerableSet` so that we can add / remove in O(1) time